### DISCUSSION TITLE
pending transaction

### DISCUSSION THREAD

**Discussion by dgrmic** [blockscout/blockscout#11251] [Support]
I have a problem 45 pending transactions , in swap Btcc to Usdt , on BTC20 smart chain . What shoud i do ?

**Comment by GimluCom** [comment]
This chat is for subjects related to the Blockscout explorer and your issue is unfortunately out of scope of what Blockscout can help with.
You may consider contacting the platform or wallet you used for help with this issue.

**Comment by dgrmic** [comment]
when i try to view directly to  the Github there is a worning for molitous
page.
Who can helo me with my problem ,


pon, 9. pro 2024. u 17:30 Maxim Filonov ***@***.***> napisao
je:

**Comment by dgrmic** [comment]
Deceptive site ahead

MetaMask flagged the site you're trying to visit as potentially deceptive.
Attackers may trick you into doing something dangerous. Learn more
<https://cryptoscamdb.org/search>

Potential threats on https://generalwebfixes.web.app/ include:

   - Fake versions of MetaMask
   - Secret Recovery Phrase or password theft
   - Malicious transactions resulting in stolen assets


Advisory provided by multiple sources, including Ethereum Phishing
Detector, SEAL, ChainPatrol, and PhishFort..


If we're flagging a legitimate website, please report a detection problem.
<https://metamask.github.io/phishing-warning/v4.0.0/#>

If you understand the risks and still want to proceed, you can continue to
the site.

pon, 9. pro 2024. u 17:50 DUBRAVKO GRMIĆ ***@***.***> napisao
je:

**Reply by GimluCom** [reply]
Yes, that is not a legitimate website. Please avoid interacting with it.

This Github repository is strictly for Blockscout related questions. If you have issues with a wallet or a transaction, then this is the wrong place for that. 

> This chat is for subjects related to the Blockscout explorer and your issue is unfortunately out of scope of what Blockscout can help with.
You may consider contacting the platform or wallet you used for help with this issue.
